src/components/pages/LandingPrograms/CoderSitters.js

```diff
@@ -8,50 +8,52 @@ export default function CoderSitters() {
   return (
     <Layout>
       <Content className="pl__na__sitter__container">
-        <div className="pl__na__top__sitter__container">
-          {/* CODERSITTERS TITLE */}
-          <Title className="pl__na__sitter__title">CODERSITTERS</Title>
-          {/* CODERSITTERS INTRO PARAGRAPH */}
-          <Text className="pl__na__intro__text">
-            Parents get their needed
-            <br />
-            distraction-free time. Kids get to
-            <br />
-            learn coding through play.
-          </Text>
-          {/* BABYSITTER + INSTRUCTOR */}
-          <div className="pl__na__top__mid__container">
-            <Title className="pl__na__sitter__title__container">
-              THINK BABY SITTER + CODING INSTRUCTOR.
+        <div className="pl__na__sitter__container_wrapper">
+          <div className="pl__na__top__sitter__container">
+            {/* CODERSITTERS TITLE */}
+            <Title className="pl__na__sitter__title" level={2}>
+              CODERSITTERS
             </Title>
-
-            {/* INFO PARAGRAPH */}
-            <Text className="pl__na__sitter__text__container">
-              Our lessons include playing age-appropriate
-              <br />
-              coding games online, in tandem with a wide
+            {/* CODERSITTERS INTRO PARAGRAPH */}
+            <Text className="pl__na__intro__text">
+              Parents get their needed
               <br />
-              range of interactive offline activities. These
+              distraction-free time. Kids get to
               <br />
-              might include: drawing, prototyping, crafting
-              <br />
-              or imaginative brainstorming
+              learn coding through play.
             </Text>
-          </div>
-          {/* AGE */}
-          <div className="pl__na__age__container">
-            <Text className="pl__na__age">Ages 6 and Up.</Text>
-          </div>
+            {/* BABYSITTER + INSTRUCTOR */}
+            <div className="pl__na__top__mid__container">
+              <Title className="pl__na__sitter__title__container" level={2}>
+                THINK BABY SITTER + CODING INSTRUCTOR.
+              </Title>
+
+              {/* INFO PARAGRAPH */}
+              <Text className="pl__na__sitter__text__container">
```

Comment:
In PR reviews Edd mentioned there are ways to get around using explicit <br/> tags by using relative units for sizing within text containers.